plugins/kubernetes/app/models/kubernetes/role.rb

```diff
@@ -19,5 +19,13 @@ def label_name
     def ram_with_units
       "#{ram}Mi" if ram.present?
     end
+
+    def has_service?
+      service_name.present?
```

Comment:
We have a small inconsistency here. Currently even if a service is defined in the template, but the user does not specify the name when creating a role, we will assume that no service should be deployed. It would be better to pre-populate this from the template, or solely rely on the template.